### PR TITLE
docs: 📚 Add Nuxt option information for extending theme

### PR DESCRIPTION
**Pull request by carwack**
## Description
Added a small part under the use of 'extending-theme' for the Nuxt.js configuration.

## Motivation and Context
I was missing this information when using the Theme Object in my Nuxt.js project

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: website/pages/extending-theme.mdx [hunk 1]
```diff
@@ -61,3 +61,22 @@ new Vue({
...
+  ...
+  modules: [
+    '@chakra-ui/nuxt',
+    '@nuxtjs/emotion'
+  ],
+  chakra: {
+      extendTheme: customTheme
+    }
+  }
+  ...
```
Comment (codebender828) [inline website/pages/extending-theme.mdx:80]: Thanks for the contribution toward this. Do you think that it's probably better to reduce the amount of boiler plate in the `nuxt.config.js` code snippet here? So we'd only leave the chakra option.
```suggestion
  // ...
  chakra: {
      extendTheme: customTheme
    }
  }
  // ...
```

**Review by codebender828 — CHANGES_REQUESTED** [review]

File: website/pages/extending-theme.mdx (hunk 1 above)
Comment (carwack) [reply website/pages/extending-theme.mdx:80]: Yes, it will be clear enough, but then I do think the last } has to be deleted too isn't it?

Comment (dombavetta) [reply website/pages/extending-theme.mdx:80]: Agreed on the reduced boilerplate, we already cover the setup in the nuxt.js section so I don't think it's necessary.

Comment (codebender828) [reply website/pages/extending-theme.mdx:80]: @carwack Yup! Good catch with the closing brace. Was a typo. 👍🏽

Comment (carwack) [reply website/pages/extending-theme.mdx]: @codebender828 @dombavetta Thanks for the input! I did refactor the text. 😄

**Review by codebender828 — APPROVED** [review]
Thank you @carwack

**Pull request merged**